packages/react-card/src/components/CardHeader/CardHeader.types.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import type { ComponentPropsCompat, ComponentStateCompat, ShorthandPropsCompat } from '@fluentui/react-utilities';
+
+/**
+ * CardHeader Props
+ */
+export interface CardHeaderProps extends ComponentPropsCompat, React.HTMLAttributes<HTMLElement> {
+  /**
+   * Image slot
+   */
+  image: ShorthandPropsCompat<React.ImgHTMLAttributes<HTMLImageElement>>;
```

Comment:
Should the image be required?

Reply:
Yes, at least for now. We don't really have any designs at the moment for scenarios with missing pieces and making it optional in the future wouldn't bring breaking changes.